test/rubocop/cop/netlify/invalid_model_assignment_test.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class InvalidModelAssignmentTest < Minitest::Test
+  def test_offense_with_model_assignment
+    assert_offense <<~RUBY
+      class User
+        attr_reader :attributes
+      end
+      user = User.new
+      user.attributes[:email] = \"test@example.com\"
```

Comment:
[dust] this escape... is maybe not necessary? (sorry if that's actually required, I'm definitely not familiar with 👮‍♂️ )

Reply:
It was to match the annotated code when the offense happens, but I tried without and it still works.

Reply:
make sense, thanks for explaining & thanks for updating!